controls/pcidss_4_ocp4.yml

```diff
@@ -374,33 +374,60 @@ controls:
         immediately after a system component is connected to a production environment.
       levels:
         - base
-      status: pending
+      status: partial
       notes: |-
-        Interestingly this requirement recommends other standards, such as Center for Internet
-        Security (CIS), International Organization for Standardization (ISO), National Institute
-        of Standards and Technology (NIST), Cloud Security Alliance, and product vendors. So, the
-        rules included here are very generic in terms of hardening.
-      rules: []
+        The Compliance Operator provides an automated way of checking for a secure configuration
+        of OpenShift according to known security standards, as well as automatically remediating
+        deviations from the target configuration.
+        It's usage is recommended to ensure appropriate configuration and is set and prevent drift.
+
+        System components outside of OpenShift cannot be checked by the Compliance Operator.
+
+        This control is also addressed by applying the OpenShift CIS recommendations.
+      rules:
+        - scansettingbinding_exists
```

Comment:
Ok - so we're just ensuring a least some scan is running?

Reply:
Yes. I ported this over from requirement PCI-DSS v3.2.1 Req 2.2.4.

My interpretation is that CO can help apply and maintain OCP's configuration.